tests/runtime/probe

```diff
@@ -68,17 +68,18 @@ REQUIRES_FEATURE fentry
 REQUIRES_FEATURE btf
 AFTER ./testprogs/fentry_args 1111 2222
 
-NAME fentry_warn_missing_probes
+NAME fentry_missing_probes
 PROG fentry:vfs_read,fentry:nonsense { exit(); }
-EXPECT WARNING: No BTF found for nonsense, skipping.
+EXPECT ERROR: No BTF found for nonsense.
+WILL_FAIL
 
 NAME fentry_ignore_missing_probes
 PROG config = { missing_probes = "ignore" } fentry:vfs_read,fentry:nonsense { exit(); }
-EXPECT_NONE WARNING: No BTF found for nonsense, skipping.
+EXPECT_NONE WARNING: No BTF found for nonsense.
 
 NAME fentry_error_missing_probes
 PROG config = { missing_probes = "error" } fentry:vfs_read,fentry:nonsense { exit(); }
-EXPECT ERROR: No BTF found for nonsense
+EXPECT ERROR: No BTF found for nonsense.
```

Comment:
We should change this one to use `missing_probes = "warn"` to cover all values of the config.